build/lib/reclaim_python_sdk/reclaim.py

```
@@ -85,12 +76,26 @@ async def verify_proof(proof: Union[Proof, List[Proof]]) -> bool:
+            
+            if witness_url == "manual-verify":
+                witnesses.append(witness_id)
+            else:
+                witnesses = await get_witnesses_for_claim(
+                    proof.claimData.epoch, proof.identifier, proof.claimData.timestampS
+                )
         else:
-            witnesses = await get_witnesses_for_claim(
-                proof.claimData.epoch, proof.identifier, proof.claimData.timestampS
-            )
+            logger.info(f"No witnesses found for proof")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)